lib/NetworkManager.js

```diff
@@ -569,12 +569,16 @@ function generateRequestHash(request) {
   };
 
   if (!normalizedURL.startsWith('data:')) {
-    const headers = Object.keys(request.headers);
-    headers.sort();
-    for (const header of headers) {
```

Comment:
can we please minimize the change? I believe you can add a `header.toLowerCase()` as the first line in the loop:
```js
for (let header of headers) {
  header = header.toLowerCase();
  if (key === 'accept' || key === 'referer' ...)
}
```